### PR TITLE
Add TM for Formal Motoko 

### PR DESCRIPTION
**Pull request by aterga**
Screenshot 2022-10-25 at 18 22 27

### CODE REVIEW

File: themes/FormalMotoko.tmTheme [hunk 1]
```diff
@@ -0,0 +1,636 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>author</key>
+	<string>Martin Kühl</string>
+	<key>comment</key>
+	<string>Based on the Quiet Light theme for Espresso by Ian Beck.</string>
```
Comment (aterga) [inline themes/FormalMotoko.tmTheme:8]: Original theme taken from https://github.com/microsoft/vscode-textmate/blob/main/test-cases/themes/QuietLight.tmTheme

The only addition so far is `keyword.other.verification.contracts`

**Review by rvanasa — APPROVED** [review]
Cool! Maybe we could also add color overrides for the built-in VS Code themes.

**Comment by rvanasa** [comment]
Screen Shot 2022-10-25 at 11 46 08 AM

What do you think of using standard syntax highlighting similar to `assert` in other languages? It's not as distinctive, but it would work for any VS Code theme.

**Comment by aterga** [comment]
This looks very nice, but would we be able to distinguish dynamic and static assertions? Ideally, they shouldn't look too similar.

**Comment by rvanasa** [comment]
Dynamic assertions are apparently highlighted in a non-standard way in the current TextMate grammar, so there is a clear distinction.

**Comment by aterga** [comment]
Okay, but then the colors match those of `if` and `else`, which might also be sub-optimal.

**Comment by rvanasa** [comment]
That's true, and it also seems reasonable to give these formal verification contracts a distinct appearance. 

Perhaps we could directly use the themes and conventions from the Viper extension? It looks like these grammar changes automatically work with `Viper-Light` and `Viper-Dark`. Since we include `viper-ide` as an extension dependency, both of these themes will always be available. This would also simplify the process of supporting more themes over time (for both Viper and Formal Motoko), which would be great from a UX standpoint.

**Comment by aterga** [comment]
That's a very good idea. At a later point in time, maybe we could even factor out the themes into a language-independent, verification-oriented theme bundle (which could become a common dependency of both `vscode-motoko` and `viper-ide`.

**Comment by rvanasa** [comment]
Sounds great! Feel free to merge (with or without the custom theme for now).

**Pull request merged**